src/main/java/com/checkmarx/flow/service/JiraService.java

```diff
@@ -1131,14 +1131,14 @@ private void closeIssueInCaseNotWithinResults(ScanRequest request, Map<String, S
             try {
                 if (!map.containsKey(jiraIssue.getKey()) && (request.getBugTracker().getOpenStatus().contains(jiraIssue.getValue().getStatus().getName()))) {
                     /*Close the issue*/
-                    log.info("Closing issue with key {}", jiraIssue.getKey());
+                    log.info("Closing issue "+ jiraIssue.getValue().getKey()+" with key {}", jiraIssue.getKey());
                     this.transitionCloseIssue(jiraIssue.getValue().getKey(),
                             request.getBugTracker().getCloseTransition(), request.getBugTracker(), false); //No false positives
                     closedIssues.add(jiraIssue.getValue().getKey());
 
                 }
             } catch (HttpClientErrorException e) {
-                log.error("Error occurred while processing issue with key {}", jiraIssue.getKey(), e);
+                log.error("Error occurred while processing issue "+ jiraIssue.getValue().getKey()+" with key {}", jiraIssue.getKey(), e);
```

Comment:
Please use {} to introduce additional information in a log entry.  ie. log.info("{} - {}", a, b);